### PR TITLE
[member #2] Add options

### PR DESCRIPTION
**Pull request by RaphDasilva**
## Did the following:

- Added the  following options in `app.rb`:
  - `list_music_albums`
  - `list_genres` (e.g 'Comedy', 'Thriller')
  - `add_music_album`

### CODE REVIEW

File: bin/app.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    @genres = []
+  end
+
+  def add_music_album(name, on_spotify, publish_date)
+    music_album = MusicAlbum.new(name, on_spotify, publish_date)
+    @music_album << music_album
+  end
+
+  def list_music_albums
+    @music_album.each { |music_album| puts music_album }
```
Comment (anthonymr) [inline bin/app.rb:15]: To be able to "puts" an object you need to implement the "to_s" special method inside the object. In this case, you need to add a to_s method to the `MusicAlbum` class.

This is the `to_s` method for `Book` class:

```ruby
  def to_s
    "(#{@id}) publisher: #{@publisher} - cover state: #{@cover_state} - publish date: #{@publish_date.to_date}"
  end
```

File: bin/music_album.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :name
+  attr_reader :publish_date
```
Comment (anthonymr) [inline bin/music_album.rb:4]: You don't need to add this line, due this is already specified in the super class.

File: bin/music_album.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+  def initialize(name, on_spotify)
+    super(publish_date)
+    @name = name
```
Comment (anthonymr) [inline bin/music_album.rb:8]: According to the diagram, MusicAlbum class does not have a `name` attribute.

File: bin/music_album.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    @publish_date = publish_date
```
Comment (anthonymr) [inline bin/music_album.rb:9]: @publish_date is already set by `super()` constructor, you can delete this line.

**Review by anthonymr — COMMENTED** [review]
Hi @AbdimulhinYussuf3675, good job so far. please, can you make those changes?

**Review by AbdimulhinYussuf3675 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi @RaphDasilva  
Your project is complete! There is nothing else to say other than... Great job 👏🏾  Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Implemented changes requested :white_check_mark:
  * Good Job :+1: :smile: 
 
 Now you can merge this branch and move on 🥇 
 
 ## Optional suggestions

- [x] N/A keep doing the great work :wink: 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @AbdimulhinYussuf3675 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**